supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -52,22 +61,41 @@ class JesInitializationActor(override val workflowDescriptor: BackendWorkflowDes
   /**
     * Abort all initializations.
     */
+  // TODO PBE ??? is not cool
   override def abortInitialization(): Unit = ???
```

Comment:
We do seem to eat exceptions in some cases.  In the process of writing this I had done something boneheaded which caused one of these lifecycle actors to throw, but there was not a whiff of it in the logs.